azure-pipelines/official.yml

```diff
@@ -62,6 +62,8 @@ extends:
           exclusionsFile: $(System.DefaultWorkingDirectory)\azure-pipelines\PoliCheckExclusions.xml
         suppression:
           suppressionFile: $(System.DefaultWorkingDirectory)\azure-pipelines\falsepositives.gdnsuppress
+        sbom:
+          enabled: ${{ or(parameters.RealSign, eq(variables['Build.Reason'],'Schedule')) }} # Disable the generation for SBOMs for artifacts in unsigned builds since it's slow
```

Comment:
Unless SBOM is off by default in the Unofficial templates, can you make the change there too?